docs/src/docs/asciidoc/end-to-end-maven-guide.adoc

```diff
@@ -45,31 +46,31 @@ For convenience, you can create a Maven profile and add the plugin into it:
 [source,xml, role="multi-language-sample"]
 ----
 <profiles>
-  <profile>
-    <id>native</id>
-    <build>
-      <plugins>
-        <plugin>
-          <groupId>org.graalvm.buildtools</groupId>
-          <artifactId>native-maven-plugin</artifactId>
-          <version>${native.maven.plugin.version}</version>
-          <extensions>true</extensions>
-          <executions>
-            <execution>
-              <id>build-native</id>
-              <goals>
-                <goal>compile-no-fork</goal>
-              </goals>
-              <phase>package</phase>
-            </execution>
-          </executions>
-          <configuration>
-            <mainClass>org.example.Main</mainClass>
-          </configuration>
-        </plugin>
-      </plugins>
-    </build>
-  </profile>
+    <profile>
+      <id>native</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.graalvm.buildtools</groupId>
+            <artifactId>native-maven-plugin</artifactId>
+            <version>{maven-plugin-version}</version>
```

Comment:
I think the use of a property was intentional and a good Maven practice. This would inline the version in the samples.